app/Http/Controllers/ProfileController.php

```diff
@@ -151,7 +151,7 @@ public static function getValidations()
                 'nullable',
                 // Discord user name validation: https://discord.com/developers/docs/resources/user
                 // (simplified because we don't need to check all edge cases)
-                'regex:/^[^@#:]{2,32}#[0-9]{4}$/',
+                'regex:/^[a-z0-9_\.\#]{2,32}$/i',
```

Comment:
should be sufficient to allow hash instead of restricting the pattern to just alpha-numerical strings

Reply:
done